apollo-compiler/src/main/kotlin/com/apollostack/compiler/ir/FragmentTypeSpecBuilder.kt

```diff
@@ -1,14 +1,21 @@
 package com.apollostack.compiler.ir
 
+import com.apollostack.compiler.TypeSpecBuilder
 import com.apollostack.compiler.convertToPOJO
 import com.squareup.javapoet.TypeSpec
 import javax.lang.model.element.Modifier
 
 class FragmentTypeSpecBuilder(
     val fragment: Fragment,
-    val generateClasses: Boolean
-) : CodeGenerator {
-  override fun toTypeSpec(): TypeSpec = fragment.toTypeSpec().let {
+    val generateClasses: Boolean,
+    val fragmentPkgName: String,
+    val irPkgName: String
+) : TypeSpecBuilder {
+  override fun packageName(): String {
+    return fragmentPkgName
+  }
+
+  override fun toTypeSpec(): TypeSpec = fragment.toTypeSpec(irPkgName).let {
```

Comment:
Always have a question for that, should it be on the next line when you use `=` or not ? :)